release/long_running_tests/workloads/serve_failure.py

```diff
@@ -172,5 +172,6 @@ def run(self):
                 break
 
 
-tester = RandomTest(max_deployments=NUM_NODES * CPUS_PER_NODE)
+random_killer = RandomKiller.remote()
```

Comment:
What's the effect of pulling it out of RandomTest?

Reply:
It doesn't affect the test's behavior. I did it based on [@edoakes's feedback](https://github.com/ray-project/ray/pull/31945#discussion_r1088008801) from the previous change.